### PR TITLE
fix compile error on ubuntu16.04

### PR DESCRIPTION
**Pull request by libangxiang**
This is to fix the following compile error on ubuntu16.04 build:

```
src/gui/src/mainWindow.cpp: In member function ‘void gui::MainWindow::createActions()’:
src/gui/src/mainWindow.cpp:194:56: error: no matching function for call to ‘QAction::QAction(const char [20])’
   congestion_setup_ = new QAction("Congestion Setup...");
   
src/gui/src/displayControls.h:65:8: error: ‘function’ in namespace ‘std’ does not name a template type
std::function<void(bool)> action;
```

### CODE REVIEW

File: src/gui/src/mainWindow.cpp [hunk 1]
```diff
@@ -191,7 +191,7 @@ void MainWindow::createActions()
   zoom_out_ = new QAction("Zoom out", this);
   zoom_out_->setShortcut(QString("Shift+Z"));
 
-  congestion_setup_ = new QAction("Congestion Setup...");
+  congestion_setup_ = new QAction("Congestion Setup...", this);
```
Comment (maliberty) [inline src/gui/src/mainWindow.cpp:194]: What version of Qt are you using?  This should match:

QAction::QAction(const QString &text, QObject *parent = nullptr)
See https://doc.qt.io/qt-5/qaction.html#QAction-1

Comment (maliberty) [reply src/gui/src/mainWindow.cpp:194]: Normally the parent is used for a QActionGroup.

Comment (bxli2015) [reply src/gui/src/mainWindow.cpp]: Hi Matt, thanks for the review. 

My qt is v5.5.1, and I corrrected the fix from "this" to nullptr.

```
$> qmake -v
QMake version 3.0
Using Qt version 5.5.1 in /usr/lib/x86_64-linux-gnu
```

**Review by maliberty — APPROVED** [review]

**Comment by maliberty** [comment]
@tspyrou to merge

**Comment by maliberty** [comment]
merged to private

**Comment by maliberty** [comment]
I'm not sure why this didn't auto-close but I see the change is in the public repo.

**Pull request closed without merging**